code/modules/mob/living/carbon/xenomorph/xeno_defines.dm

```diff
@@ -215,6 +215,12 @@
 	/// The additional amount of stacks that the Sentinel will apply on eligible abilities.
 	var/additional_stacks = 0
 
+	// *** Behemoth Abilities ***
+	/// The maximum amount of Wrath that we can have stored at a time.
+	var/wrath_max = 0
+	/// The icon state of the Wrath bar.
+	var/wrath_icon_state = "wrath"
```

Comment:
This is not changed anywhere no idea why its a var

Reply:
This is because the Wrath bar is separate from health and plasma, not unlike a third bar. This is necessary so as to define the icon for that bar, because it needs specific DMI offsets to locate it besides the other bars.

Reply:
I see what you mean. Donezo.